package-lock.json

```diff
@@ -1,10355 +1,8 @@
 {
     "name": "avatarcropper",
     "version": "1.0.0",
-    "lockfileVersion": 2,
+    "lockfileVersion": 1,
```

Comment:
Please don't delete package lock :(

Reply:
i didn't change that myself, something else must've changed it